src/main/webapp/WEB-INF/tags/fp/corporateBilledCorporatePaidTransactions.tag

```diff
@@ -0,0 +1,98 @@
+<%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
+
+<%@ attribute name="editingMode" required="false" type="java.util.Map"%>
+<%@ attribute name="editableAccounts" required="true" type="java.util.Map"
+              description="Map of Accounts which this user is allowed to edit" %>
+<%@ attribute name="editableFields" required="false" type="java.util.Map"
+              description="Map of accounting line fields which this user is allowed to edit" %>
+
+<c:set var="columnCountUntilAmount" value="8" />
+<c:set var="columnCount" value="${columnCountUntilAmount + 1 + (KualiForm.documentActions[Constants.KUALI_ACTION_CAN_EDIT] ? 1 : 0)}" />
+<c:set var="accountingLineAttributes" value="${DataDictionary['TargetAccountingLine'].attributes}" />
+
+<kul:tab tabTitle="Accounting Lines" defaultOpen="true" tabErrorKey="${KFSConstants.TARGET_ACCOUNTING_LINE_ERROR_PATTERN},document.transactionEntries*"
+         helpUrl="${KualiForm.accountingLineImportInstructionsUrl}" helpLabel="Import Templates">
+  <c:set var="transactionAttributes" value="${DataDictionary.ProcurementCardTransactionDetail.attributes}" />
+  <c:set var="vendorAttributes" value="${DataDictionary.ProcurementCardVendor.attributes}" />
+  <c:set var="cardAttributes" value="${DataDictionary.ProcurementCardHolder.attributes}" />
```

Comment:
There are numerous class and/or DD references in this file that are referencing PCDO objects instead of CBCP ones. To reduce the likelihood of errors, I would suggest changing these references to use the CBCP objects when possible. (If you're still using PCDO vendor instead of a new CBCP vendor, then that one can stay as-is.)

Reply:
I was able to switch to CorporateBilledCorporatePaidTransactionDetail.  But we make reuse of ProcurementCardVendor and ProcurementCardHolder.  Do you think we should avoid that reuse, and make no objects, and database attributes for these two as well?

Reply:
If you think that the reuse of PCard vendor and holder on the CBCP docs would cause too much confusion or contention with the PCDO docs, then I would suggest creating some separate CBCP versions of them. Those two tables already have doc number as part of the primary key, so having CBCP-related items in those tables shouldn't interfere with the PCDO docs, but it would still require checking the doc number to see what type of document they're associated with.

Reply:
OK, I will move these two out to CBCP objects and tables